src/bic/router/middleware/login.js

```diff
@@ -0,0 +1,47 @@
+define(function (require) {
+  'use strict';
+
+  // local modules
+
+  var c = require('bic/console');
+  var API = require('bic/api');
+  var $ = require('jquery');
+  var _ = require('underscore');
```

Comment:
I'm not sure this is worth holding up this PR, but...

`$` and `_` are not "local" modules, they're "foreign". So pedantically we should either remove the section comments, or rearrange the `require()`s so they are consistently arranged as they are in other modules with section comments.